src/MergeContext.js

```diff
@@ -236,9 +236,12 @@ class MergeContext {
         assert(this._tagSha);
         this._log("finish merging...");
         try {
+            await GH.removeProtectedBranchPullRequestReviewEnforcement(this._prBaseBranch());
```

Comment:
I was under impression that we do not want the bot to change branch protections. The PR description does not mention these changes. Remove?